schema/divisions/division.yaml

```diff
@@ -51,6 +51,37 @@ properties:     # JSON Schema: Top-level object properties.
           Romansh.
         allOf:
           - "$ref": "../defs.yaml#/$defs/propertyDefinitions/commonNames"
+      class:
+        description:
+          Local name for the subtype property, in English.
+
+          For example, the Danish city of Roskilde has the subtype
+          'locality', but in the local administrative hierarchy it is
+          referred to as a 'city'.
+
+          This property relies on existing local_type values, as well
+          as constructed values like 'megacity', for example.
+      prominence:
+        description:
+          An integer value, representing Overture's view of
+          the locality's significance. This value can be used to help
+          drive cartographic display of localities and is derived
+          from various factors including but not limited to
+          population, capital status, and type. A lower value
+          indicates a high significance or importance.
+      is_rendering:
+        description:
+          A boolean, 0 or 1, to indicate whether or not the feature
+          geometry is intended for map rendering and cartographic
+          display purposes. This flag is typically set to 1 on
+          land-based features and boundaries that are clipped to land.
+      is_processing:
+        description:
```

Comment:
- Booleans are `true` or `false` in JSON schema!
- I think these are meant to be on division area/boundary?
- We should push ourselves a bit harder on the naming.